### PR TITLE
[Improvement] Support Torchvision transformations

### PR DESCRIPTION
**Pull request by kennymckormick**
## Motivation

Use torchvision transformations directly in mmaction2.

## Modification

Add a new pipeline component: TorchvisionTrans.
Modify Compose code.
Remove 3 deprecated components: EntityBoxCrop, EntityBoxFlip, EntityBoxRescale

## Drawback

This feature is just used for prototype verification (calling torchvision transformations can be much slower in the preprocessing pipeline).

### CODE REVIEW

File: tests/test_data/test_compose.py [hunk 1]
```diff
@@ -4,6 +4,15 @@
...
 from mmaction.datasets.pipelines import Compose, ImageToTensor
 
+try:
+    import torchvision
+    from distutils.version import LooseVersion
+    torchvision_ok = False
+    if LooseVersion(torchvision.__version__) >= LooseVersion('0.8.0'):
+        torchvision_ok = True
+except (ImportError, ModuleNotFoundError):
+    torchvision_ok = False
```
Comment (dreamerlin) [inline tests/test_data/test_compose.py:14]: No need to try-except

Comment (dreamerlin) [reply tests/test_data/test_compose.py:14]: Alternatives

```python
TORCHVISION_VERSION = tuple(int(x) for x in torchvision.__version__.split('.')[:3])

def obsolete_torchvision_version(torchvision_version, version_threshold):
    return torchvision_version<= version_threshold
```

File: mmaction/datasets/pipelines/augmentations.py [hunk 2]
```diff
@@ -51,6 +53,43 @@ def _init_lazy_if_proper(results, lazy):
         assert 'lazy' not in results, 'Use Fuse after lazy operations'
 
 
+@PIPELINES.register_module()
+class TorchvisionTrans:
+    """Torchvision Augmentations, under torchvision.transforms.
+
+    Args:
+        type: The name of the torchvision transformation.
```
Comment (dreamerlin) [inline mmaction/datasets/pipelines/augmentations.py:61]: type(str)

File: tests/test_data/test_compose.py (hunk 1 above)
Comment (kennymckormick) [reply tests/test_data/test_compose.py:14]: try except is needed since torchvision should not be added to known-third-parties

**Review by dreamerlin — APPROVED** [review]

**Pull request merged**